src/translations/en.json

```diff
@@ -118,25 +119,49 @@
     "sign_in.password.placeholder": "Password",
     "sign_in.username.placeholder": "Username",
 
-    "shipment.add.header": "New Shipment",
+    "shipment": "Shipment",
+    "shipment.add.header": "Back to Shipment",
+    "shipment.add_new.label": "New Shipment",
+    "shipment.add_order_to_shipment.label": "Add Order to Shipment",
+    "shipment.customer_images.label": "Customer images",
     "shipment.customer_name.label": "Customer",
+    "shipment.customer_name_table.label": "Customer Name",
```

Comment:
Repeated: `"customer_type.column.name": "Customer Name",` and there is more similar thing. We need to decide if we use what we already added or every time add new translations depends on page where we use it :)